khl/Bot.py

```diff
@@ -1,13 +1,17 @@
 import shlex
```

Comment:
making a new file `BotPreview.py` rather making new functions with `..._preview()` seems better to me